app/src/components/trace/SpanAnnotationsEditor.tsx

```diff
@@ -131,62 +132,73 @@ function NewAnnotationButton(props: NewAnnotationButtonProps) {
     spanNodeId,
     onAnnotationNameSelect,
   } = props;
-  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+  const [popoverRef, setPopoverRef] = useState<HTMLDivElement | null>(null);
   return (
-    <PopoverTrigger
-      placement="bottom end"
-      crossOffset={300}
-      isOpen={isPopoverOpen}
-      onOpenChange={(isOpen) => {
-        setIsPopoverOpen(isOpen);
-      }}
-    >
-      <TriggerWrap>
+    <>
+      <DialogTrigger>
         <Button
           variant={disabled ? "default" : "primary"}
           isDisabled={disabled}
           size="S"
           leadingVisual={<Icon svg={<Icons.PlusCircleOutline />} />}
-          onPress={() => {
-            setIsPopoverOpen(true);
-          }}
         >
-          New Annotation
+          Add Annotation
```

Comment:
I'd probably start thinking of moving this to the top. Just a suggestion going forward

Reply:
Agree